File: 001/C/solution.c
#include <stdio.h>

int		resolve(void)
{
	int sum = 0;

	for (int i = 1; i < 1000; ++i)
		if (i % 3 == 0 || i % 5 == 0)
			sum += i;
	return (sum);
}

int		main(void)
{
	printf("%d\n", resolve());
	return (0);
}

File: 001/Python/solution.py

def resolve() -> int:

    predicate = lambda i: i % 3 == 0 or i % 5 == 0

    naturals = range(1, 1000)

    multiples = filter(predicate, naturals)

    return sum(multiples)


if __name__ == "__main__":

    print(resolve())

